### PR TITLE
chore (refs T30980): add changelog.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T30980

- remove some component class, to pass it from outside.

https://github.com/demos-europe/demosplan-core/pull/1604

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ## UNRELEASED
 
+### Added
+- ([#325](https://github.com/demos-europe/demosplan-ui/pull/325)) remove some component class, to pass it from outside. ([@ahmad-demos](https://github.com/ahmad-demos))
```
Comment (salisdemos) [inline CHANGELOG.md:8]:
```suggestion
- ([#325](https://github.com/demos-europe/demosplan-ui/pull/325)) **BREAKING** remove spacing classes from DpAnonymizeText. ([@ahmad-demos](https://github.com/ahmad-demos))
```

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ## UNRELEASED
 
+### Added
+- ([#325](https://github.com/demos-europe/demosplan-ui/pull/325)) **BREAKING** remove spacing classes from DpAnonymizeText. ([@ahmad-demos](https://github.com/ahmad-demos))
```
Comment (spiess-demos) [inline CHANGELOG.md:8]: In other places, this is also not capitalized. But after it, a colon appears and the next word is written as if a sentence would start:

```suggestion
- ([#325](https://github.com/demos-europe/demosplan-ui/pull/325)) **Breaking:** Remove spacing classes from DpAnonymizeText ([@ahmad-demos](https://github.com/ahmad-demos))
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

**Review by salisdemos — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,9 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ## UNRELEASED
 
+### Added
```
Comment (spiess-demos) [inline CHANGELOG.md:7]: "Added" seems to be quite coiunter-intuitive if something has been removed.

Comment (salisdemos) [reply CHANGELOG.md:7]: or at least changed

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**